### ISSUE TITLE
incorrect tip banner text for tipping an image/video - follow up to 4548, 4583

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#5412]
## Description 
Was originally reported with #4583 however after work was done to accommodate the new Twitter layout (#4548), this issue reappeared.

The test on the tipping banner is incorrect. See https://github.com/brave/brave-browser/issues/4583#issuecomment-496248934 for additional information.

## Steps to Reproduce

   1. Locate a tweet with just an image (can use this one if needed, should work for both staging and prod: https://twitter.com/bravelaurenwags/status/1131664248856354816)
   2. Ensure you're using the new twitter layout. Click on in-line Tip button 

## Actual result:
![Screen Shot 2019-07-26 at 2 41 12 PM](https://user-images.githubusercontent.com/28145373/61974425-7eedd280-afb4-11e9-847f-5c354b9fc9f8.png)

## Expected result:
![Screen Shot 2019-07-26 at 2 36 29 PM](https://user-images.githubusercontent.com/28145373/61974431-81502c80-afb4-11e9-9b01-1bef095e6494.png)

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 0.67.117 Chromium: 76.0.3809.62 (Official Build) (64-bit)
-- | --
Revision | 7b77856b3aa34d72f246d12340fc1ded8b2c0e83-refs/branch-heads/3809@{#798}
OS | Mac OS X

## Version/Channel Information:

- Can you reproduce this issue with the current release? n/a
- Can you reproduce this issue with the beta channel? n/a (beta channel build doesn't have new twitter layout fix yet)
- Can you reproduce this issue with the dev channel? n/a (dev channel build doesn't have new twitter layout fix yet)
- Can you reproduce this issue with the nightly channel? unsure

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by kjozwiak** [comment]
CCing @mandar-brave @rebron looks like this regressed when Twitter moved to the new design. Are you guys okay releasing `0.67.x` with this issue? As @LaurenWags mentioned above, it was fixed via https://github.com/brave/brave-browser/issues/4583 and was supposed to go out this release, but it looks like the new design regressed this again.

Maybe the work @emerick is doing with https://github.com/brave/brave-browser/issues/5388 might also fix this one?

**Comment by mandar-brave** [comment]
yea; i would not block on this @kjozwiak.
cc @NejcZdovc @emerick to confirm if new fix helps. 

@kjozwiak my gut says their fix is to get twitter data access via panel. This is an issue that just got missed. I would not block release on it.

**Comment by kjozwiak** [comment]
Yup, agreed re: blocking. It was fixed via #4583 for this release, it's just that the new web design regressed it again. So technically, we haven't even released a fix as #4583 was going to go out in `0.67.x` on Tuesday.

**Comment by emerick** [comment]
Most likely due to us now using Twitter's API to retrieve tweet metadata (something we have to do with the advent of the new Twitter redesign). In the past, we walked the DOM and were able to get just the tweet text. But I think the API retrieves everything, including the image links.

**Comment by NejcZdovc** [comment]
@alexwykoff what would you like to do for this case?
